### PR TITLE
Doc-109

### PR DESCRIPTION
**Pull request by bligneri**
First pass as integrating the Rating API documentation.

### CODE REVIEW

File: dev-shipperhq/blog/2022-01-26-first-import-of-rating-api-doc.md [hunk 1]
```diff
@@ -0,0 +1,625 @@
...
+Submit a request that includes the contents of the cart, like item’s weight, value and quantity and include destination address information 
+Can include any other shipping options required like liftgate,  residential address or specific dates for delivery
+Our API will return shipping rates for the cart using your ShipperHQ configured rules and carriers
+Depending on whether you choose to request simple rates or full shipping rates, we can return basic rate information or full details of carriers, shipments, dates and shipping options available.
+
+## Retrieving Rating Information
+You can retrieve shipping rates by querying the Rating GraphQL API. 
+
+## GraphQL API URL and Limits
+- Endpoint URL: The current GraphQL API endpoint is at https://api.shipperhq.com/v2/graphql and all requests must be ```HTTP POST``` requests with ```application/JSON``` encoded bodies.
```
Comment (rachelchhay) [inline dev-shipperhq/blog/2022-01-26-first-import-of-rating-api-doc.md:37]: Code blocks use 3 backticks. If it's just one line, you can use 1 backtick. Can you make these changes throughout the page?

```suggestion
- Endpoint URL: The current GraphQL API endpoint is at https://api.shipperhq.com/v2/graphql and all requests must be `HTTP POST` requests with `application/JSON` encoded bodies.
```

File: dev-shipperhq/blog/2022-01-26-first-import-of-rating-api-doc.md [hunk 2]
```diff
@@ -0,0 +1,625 @@
...
+        "value" : "ORIGIN1,ORIGIN2"
+      } ], 
+```
+
+## Requested Options
+
+Requested Options in the requestedOptions field in the request allow you to include services for the entire shipment such as Liftgate or Residential delivery. These are useful if you are using Freight carriers or small package carriers that offer specific delivery methods for residential delivery. The most common attributes are listed below. 
+
+::: tip All values are case-sensitive
+:::
```
Comment (rachelchhay) [inline dev-shipperhq/blog/2022-01-26-first-import-of-rating-api-doc.md:157]: There's a space in front of "tip," so the styling was off. 

```suggestion
:::tip All values are case-sensitive 
:::
```

File: dev-shipperhq/blog/2022-01-26-first-import-of-rating-api-doc.md [hunk 3]
```diff
@@ -0,0 +1,625 @@
...
+
+Requested Options	Data Type	Description
+liftgate_required	Boolean	This specifies liftgate is required at the destination.
+notify_required	Boolean	This requests an appointment delivery or notice of delivery.
+inside_delivery	Boolean	This requests delivery inside the destination.
+limited_delivery	Boolean	This specifies limited access at the destination address.
+destination_type	Boolean	This specifies if a destination is a Residential or Business address and is used for freight and small package shipments.
```
Comment (rachelchhay) [inline dev-shipperhq/blog/2022-01-26-first-import-of-rating-api-doc.md:164]: We probably want to re-format this. Maybe add a table and backticks for code?

File: dev-shipperhq/blog/2022-01-26-first-import-of-rating-api-doc.md [hunk 4]
```diff
@@ -0,0 +1,625 @@
...
+Item attributes in the request allow you to include item-specific values like a shipping group or an origin. They are required if you are using any type of features such as carrier rules, dimensional shipping, multi-origin, etc. The most common attributes are listed below. 
+
+
+:::tip Values are case sensitive and require delimiter
+All values are case sensitive and require a delimiter between multiple values. The default delimiter is a comma, but if your data is already separated by hashes (#), it is required for you to set the appVersion to 2.0.0 within the siteDetails.
+
+:::
+Attribute Name	Description
+shipperhq_shipping_group	This is used to place the shipping group on an item to match carrier rules.
+shipperhq_warehouse	This is used if you are using multi-origin functionality in order to specify which origin(s) are applicable for an item. These values are separated by the “#’ character and must match origin names configured in ShipperHQ.
```
Comment (rachelchhay) [inline dev-shipperhq/blog/2022-01-26-first-import-of-rating-api-doc.md:123]: We'll want to add backticks when referencing code. 

```suggestion
`shipperhq_warehouse`	This is used if you are using multi-origin functionality in order to specify which origin(s) are applicable for an item. These values are separated by the “#’ character and must match origin names configured in ShipperHQ.
```

**Review by rachelchhay — CHANGES_REQUESTED** [review]
The page looks great so far! I added a few suggestions. You can take a look at them and select "commit suggestion." This will add my suggestion as a commit in your PR, so you don't have to make the changes yourself. 

When you go to your local env, make sure to do a `git pull` to see these changes locally.

File: dev-shipperhq/blog/2022-01-26-first-import-of-rating-api-doc.md (hunk 1 above)
Comment (bligneri) [reply dev-shipperhq/blog/2022-01-26-first-import-of-rating-api-doc.md:37]: Good call. I did replace all of them

File: dev-shipperhq/blog/2022-01-26-first-import-of-rating-api-doc.md (hunk 2 above)
Comment (bligneri) [reply dev-shipperhq/blog/2022-01-26-first-import-of-rating-api-doc.md:157]: Good call.

File: dev-shipperhq/blog/2022-01-26-first-import-of-rating-api-doc.md (hunk 3 above)
Comment (bligneri) [reply dev-shipperhq/blog/2022-01-26-first-import-of-rating-api-doc.md:164]: Yep

**Review by rachelchhay — APPROVED** [review]
Looks good!

**Pull request merged**